core-ui/src/components/Predefined/Details/Events.details.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  formatInvolvedObject,
+  formatSourceObject,
+  navigateToNamespaceOverview,
+} from 'hooks/useMessageList';
+
+import { Icon, LayoutPanel, Link } from 'fundamental-react';
+import { ReadableCreationTimestamp, Tooltip } from 'react-shared';
+import { LayoutPanelRow } from 'shared/components/LayoutPanelRow/LayoutPanelRow';
+
+const RowComponent = ({ name, value }) =>
+  value ? <LayoutPanelRow name={name} value={value} /> : null;
+
+const Message = event => {
+  const { t } = useTranslation();
+
+  return (
+    <LayoutPanel key="specification-panel" className="fd-margin--md">
+      <LayoutPanel.Header>
+        <LayoutPanel.Head title={t('events.headers.message')} />
+      </LayoutPanel.Header>
+      <LayoutPanel.Body>
+        {event.message && (
+          <RowComponent
+            name={t('events.headers.message')}
+            value={event.message}
+          />
+        )}
+        {event.reason && (
+          <RowComponent
+            name={t('events.headers.reason')}
+            value={event.reason}
+          />
+        )}
+      </LayoutPanel.Body>
+    </LayoutPanel>
+  );
+};
+
+export const EventsDetails = ({ DefaultRenderer, ...otherParams }) => {
+  const { t } = useTranslation();
+
+  const customColumns = [
+    {
+      header: t('events.headers.involved-object'),
+      value: event => formatInvolvedObject(event.involvedObject),
+    },
+    {
+      header: t('events.headers.source'),
+      value: event => formatSourceObject(event.source),
+    },
+    {
+      header: t('common.labels.namespace'),
+      value: event => (
+        <Link
+          className="fd-link"
+          data-testid="details-link"
+          onClick={() => navigateToNamespaceOverview(event.metadata.namespace)}
+        >
+          {event.metadata.namespace}
+        </Link>
+      ),
+    },
+    {
+      header: t('events.headers.type'),
+      value: event => (
+        <p>
+          {event.type}{' '}
+          {event.type === 'Warning' ? (
+            <Tooltip content={event.type}>
+              <Icon
+                ariaLabel="Warning"
+                glyph="message-warning"
+                size="s"
+                className="fd-has-color-status-2 cursor-pointer"
```

Comment:
why do we have cursor-pointer instead of a question mark as we have in other places with tooltips?

Reply:
Because I used wrong class by mistake